### DISCUSSION TITLE
where is the action of delte tc filter

### DISCUSSION THREAD

**Discussion by boost2020** [gojue/ecapture#571] [Q&A]
There is a code with this annotation in file probe_openssl_pcap.go:

```
			// customize deleteed TC filter
			// tc filter del dev eth0 ingress
			// tc filter del dev eth0 egress
			// loopback devices are special, some tc probes should be skipped
			// TODO: detect loopback devices via aquasecrity/tracee/pkg/ebpf/probes/probe.go line 322
			// isNetIfaceLo := netIface.Flags&net.FlagLoopback == net.FlagLoopback
			//	if isNetIfaceLo && p.skipLoopback {
			//		return nil
			//	}
```

So the question I want to ask is, when using the PCAP module, will the original ingress/egress tc filter be deleted first, and then the tc filter will be regenerated

**Comment by boost2020** [comment]
@cfc4n can you help me

**Reply by cfc4n** [reply]
Sorry, I forgot to reply. I‘ll have a look on the weekend. If you forget it, please remind me.

**Reply by boost2020** [reply]
Could you take a look at this for me when you have a chance @cfc4n

**Reply by cfc4n** [reply]
eCapture only clears the `tc filters` it creates itself; you can refer to https://github.com/gojue/ebpfmanager/blob/11525325c96c65722b2c2096fce73ba9122bb922/examples/clone_vs_add_hook/demo.go#L72
